src/vs/editor/common/languages.ts

```diff
@@ -1189,6 +1189,10 @@ export namespace SymbolKinds {
 		}
 		return icon;
 	}
+
+	export function asAriaLabel(label: string, kind: SymbolKind): string {
+		return `${label} Symbol: ${SymbolKinds.toIcon(kind).id.replaceAll('symbol-', '')} `;
```

Comment:
I am missing NLS, it's also rather fishy to use string-gymnastics on icon names for forming a labels

Reply:
instead of the string gymnastics I'd recommend to move this to a place that allows for reuse

https://github.com/microsoft/vscode/blob/joh/thoughtless-haddock/src/vs/workbench/contrib/codeEditor/browser/outline/documentSymbolsTree.ts#L198

Reply:
I was thinking each caller would provide their own NLS...

Reply:
I also need to reference this from the document symbols outline, not sure if that will be possible if i move it to where you suggest

Reply:
this needs to be used from other places like the `goToSymbolQuickAccess.ts`, where that cannot be referenced

Reply:
there's a `getCodiconAriaLabel` that i'm going to use instead.

Reply:
> where that cannot be referenced

Yes, that's what I meant with "move for reuse". Like, move the record with kind-2-label into `languages.ts#SymbolKinds`